recovery_backend/backend/configmap.go

```diff
@@ -69,12 +70,15 @@ func (rb *RecoveryBackend) CreateConfigMap(hostname, version string) error {
 		}
 	}
 
-	return nil
+	_, err = datastore.VerifyCreation(configMapName, "configmap", func(name string) (runtime.Object, error) {
+		return rb.Datastore.GetConfigMapRO(rb.Namespace, name, false)
+	})
+	return err
```

Comment:
Why execute this outside of `Datastore.CreateConfigMap`?

Reply:
It follows the pattern in datastore/longhorn.go. Create the resource and check the resource is ready in the lister.

Reply:
I mean, can we move this verification into `ds.CreateConfigMap()`?

Reply:
Moving the verification in CreateConfigMap is a bit of weird.
Because the CreateConfigMap updates the k8s databese directly, but the verification is verifying the data in the lister...I perfer to verify it outside of CreateConfigMap.

Reply:
Compared with asking the upper caller to take care of the cache verification, I would like to handle it inside the CRUD function. I think it's reasonable to ask the creation function to make sure the object is in-sync.

Reply:
BTW, the verifyCreation is not really necessary, because the create/update/get configmap are using k8s client api. Right?

Reply:
Yes. 
OK. Since we decide to implement https://github.com/longhorn/longhorn/issues/5263, we can skip the verification for the config map first.